### PR TITLE
Pyzhou/tdi 43717 create cosmos db sql api

### PR DESCRIPTION
**Pull request by pyzhou-talend**
https://jira.talendforge.org/browse/TDI-43717

### CODE REVIEW

File: cosmosDB/src/main/java/org/talend/components/cosmosDB/input/CosmosDBInput.java [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+    private I18nMessage i18n;
+
+    private final CosmosDBInputConfiguration configuration;
+
+    private final RecordBuilderFactory builderFactory;
+
+    private CosmosDBService service;
+
+    private DocumentClient client;
```
Comment (wwang-talend) [inline cosmosDB/src/main/java/org/talend/components/cosmosDB/input/CosmosDBInput.java]: no need to serialize, so need "transient" key. maybe not need here, but best to add, then no need to consider that thing.

File: cosmosDB/src/test/java/org/talend/components/cosmosDB/CosmosDBOutputTestIT.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    }
+
+    @Test
+    public void outputTest() {
+        config.setAutoIDGeneration(true);
+
+        CosmosDBOutput cosmosDBOutput = new CosmosDBOutput(config, service, i18n);
+        cosmosDBOutput.init();
+        Record record = createData(1).get(0);
+        System.out.println(record);
```
Comment (wwang-talend) [inline cosmosDB/src/test/java/org/talend/components/cosmosDB/CosmosDBOutputTestIT.java]: use assert?

File: pom.xml [hunk 3]
```diff
@@ -53,6 +53,7 @@
         <module>extensions</module>
         <module>pubsub</module>
         <module>netsuite</module>
+		<module>cosmosDB</module>
```
Comment (wwang-talend) [inline pom.xml]: format?

**Comment by pyzhou-talend** [comment]
@wwang-talend modified according to your comment, please help to review again.

**Pull request merged**